Formula/libbinio.rb

```diff
@@ -1,9 +1,9 @@
 class Libbinio < Formula
-  desc "Binary I/O stream class library"
-  homepage "https://libbinio.sourceforge.io/"
-  url "https://downloads.sourceforge.net/project/libbinio/libbinio/1.4/libbinio-1.4.tar.bz2"
-  sha256 "4a32d3154517510a3fe4f2dc95e378dcc818a4a921fc0cb992bdc0d416a77e75"
-  license "LGPL-2.1"
+  desc "Binary I/O stream class library (used by AdPlug)"
```

Comment:
Is it necessary to include the `(used by AdPlug)` here?

Reply:
@mywave82 I think we needn't change the description here, even though this library is used exclusively by AdPlug. We can identify that `adplug` uses it from `brew deps adplug` or `brew uses libbinio`.
```suggestion
  desc "Binary I/O stream class library"
```